video/out/wayland_common.c

```diff
@@ -1068,14 +1068,17 @@ bool vo_wayland_supported_format(struct vo *vo, uint drm_format) {
 
 /* currently unused */
 static void dmabuf_modifier(void *data,
-        struct zwp_linux_dmabuf_v1 *zwp_linux_dmabuf, uint32_t format,
-        uint32_t modifier_hi, uint32_t modifier_lo) {
-
+                            struct zwp_linux_dmabuf_v1 *zwp_linux_dmabuf,
+                            uint32_t format,
+                            uint32_t modifier_hi,
+                            uint32_t modifier_lo)
+{
```

Comment:
I don't think we need to use this many lines for the arguments here, though github's review interface doesn't let me see number of columns. In either case, format and modifier_hi look like they'd fit on the same line.